src/SingleBookingPage/bookingItem/AccordionBodyLeg.js

```diff
@@ -84,9 +85,14 @@ const AccordionBodyLeg = (props: LegProps) => {
         </div>
         <div className="guarantee">
           <Check customColor="#01bba5" height="23" />
-          <span className="text">
-            Transfer protected by the <b>Kiwi.com Guarantee</b>
-          </span>
+          <Link
+            to="/faq/RkFRQ2F0ZWdvcnk6OTM=/article/RkFRQXJ0aWNsZToxNDY="
```

Comment:
I would rather prefer `/faq/search`, see this MR on frontend:  https://gitlab.skypicker.com/frontend/frontend/merge_requests/5456

it eliminates the problem with various section